setup.cfg

```
@@ -0,0 +1,62 @@
+[metadata]
+name = pysatCDF
+version = 0.3.1
```

can point to version file with 
```suggestion
version = file: pysatCDF/version.txt
```